HRC_TXING_CHECK/HRC_Txing_Check.py

```diff
@@ -94,15 +360,29 @@
 # Inits
 #
 
+# Read the Parameter Block to SI mode definition file map
+pb_to_mode_map = PB_to_Mode_Map()
+
+# Make a list, using the pb_to_mode_map dict keys, of all the parameter blocks in
+# the SI modes that could be used when HRC is observing
+ACIS_HRC_parameter_block_list = (pb_to_mode_map.keys())
```

Comment:
Suggest this, because surrounding a `dict.keys()` does not do anything in Python 3 (verified on my own). 

```python
ACIS_HRC_parameter_block_list = list(pb_to_mode_map.keys())
```

Reply:
replaced the "list" which was accidentally removed.